python-modules/cis_identity_vault/tox.ini

```diff
@@ -15,4 +15,5 @@ deps=
   tox-run-before
   ../cis_profile
   ../cis_crypto
+  /psycopg2-2.8.3
```

Comment:
any issue with getting it from https://pypi.org/project/psycopg2/ instead? (same for the docker image build) and get the "latest version" every time?